comp/dogstatsd/listeners/udp_test.go

```diff
@@ -45,7 +45,7 @@ func newPacketPoolManagerUDP(cfg config.Component) *packets.PoolManager {
 }
 
 func TestNewUDPListener(t *testing.T) {
-	deps := fulfillDepsWithConfig(t, map[string]interface{}{})
+	deps := fulfillDepsWithConfig(t, map[string]interface{}{"dogstatsd_port": "__random__"})
```

Comment:
For unknown reason this test was complaining about port 8125 being already bound, I was not able to reproduce or figure out why, but using a random port instead (as in several other dogstatsd tests) should fix it